src/cq_cam/command.py

```diff
@@ -280,8 +345,70 @@ def to_ais_shape(self, start, as_edges=False, alt_color=False):
 
 
 class CircularCW(Circular):
-    modal = "G2"
+    modal = Path.ARC_CW.to_gcode()
 
 
 class CircularCCW(Circular):
-    modal = "G3"
+    modal = Path.ARC_CCW.to_gcode()
+
+
+class StartSequence(ConfigCommand):
+    def __init__(
+        self, spindle: Optional[int] = None, coolant: Optional[CoolantState] = None
+    ) -> None:
+        super().__init__(None, spindle, coolant)
+
+    def to_gcode(self) -> str:
+        gcode_str = CutterState.ON_CW.to_gcode()
+
+        if self.spindle != None:
+            gcode_str += f" S{self.spindle}"
+
+        if self.coolant != None:
+            gcode_str += f" {self.coolant.to_gcode()}"
+
+        return gcode_str
+
+
+class StopSequence(ConfigCommand):
+    def __init__(self, coolant: Optional[CoolantState] = None):
+        super().__init__(None, None, coolant)
+
+    def to_gcode(self) -> str:
+        gcode_str = CutterState.OFF.to_gcode()
+        if self.coolant != None:
```

Comment:
PEP-8 @ https://peps.python.org/pep-0008/#programming-recommendations

> Comparisons to singletons like None should always be done with `is` or `is not`, never the equality operators.

```suggestion
        if self.coolant is not None:
```